pages/Page.vue

```diff
@@ -1,7 +1,158 @@
-<script lang="ts" setup></script>
+<script lang="ts">
+//var page_name="something"
+//ask taz about type/data declaration
+//export default 
+//onMounted(() => {
+
+//})
+type Page ={
+    Cuid: String,
+    url : String,
+    page_name: String,
+    day_of_birth: Date,
+    day_of_passing: Date,
+    visitation_date: Date,
+    visitation_location: String,
+    timezone: String,
+    visitation_description: String,
+    funeral_date: Date,
+    funeral_description: String,
+    funeral_location: String,
+    obituary: String,
+    deadline: Date,
+    donation_goal: Number,
+    donated_amount: Number,
+    donated_percentage: Number,
+    query: String
+
+}
+const data = ref<Page>( {
+    Cuid : "",
+    url : "",
+    page_name: "",
+    day_of_birth: new Date(),
+    day_of_passing: new Date(),
+    visitation_date: new Date(),
+    visitation_location: "",
+    timezone: "",
+    visitation_description: "",
+    funeral_date: new Date(),
+    funeral_description: "",
+    funeral_location: "",
+    obituary: "",
+    deadline: new Date(),
+    donation_goal: 0,
+    donated_amount:0,
+    donated_percentage: 0,
+    query: ""
+}) 
+const create_checkup_session =async () => {
```

Comment:
You will need a variable to hold the donation form data, and that viarabel will be sent to the backend. See ofeks work in the EditPage/[id].vue file as an example

Reply:
Also note that you will need to implementing loading of the page data based on the route param